packages/dm-core-plugins/src/generic-list/GenericTablePlugin.tsx

```diff
@@ -98,121 +156,181 @@ export const GenericTablePlugin = (
       .finally(() => setIsSaveLoading(false))
   }
 
+  function openItemAsTab(item: TGenericObject, index: number) {
+    const view = { label: item?.data?.name, type: 'ViewConfig' }
+    onOpen(crypto.randomUUID(), view, `${idReference}[${index}]`)
+  }
+
+  function expandItem(index: number) {
+    const itemsCopy = [...items]
+    itemsCopy[index].expanded = !itemsCopy[index].expanded
+    setItems(itemsCopy)
+  }
+
+  function getColumnsLength() {
+    let amount = config.columns?.length || 0
+    if (functionality.delete) amount += 1
+    if (functionality.openAsTab || functionality.openAsExpandable) amount += 1
+    if (functionality.sort) amount += 1
+    return amount
+  }
+
   if (error) throw new Error(JSON.stringify(error, null, 2))
   if (loading) return <Loading />
 
   return (
-    <div
-      style={{
-        display: 'flex',
-        flexDirection: 'column',
-        width: '100%',
-      }}
-    >
+    <Stack spacing={1}>
       <Table style={{ width: '100%' }}>
         <Table.Head>
           <Table.Row>
-            {internalConfig.editMode && (
-              <Table.Cell aria-label="Controls"></Table.Cell>
+            {(functionality?.openAsTab || functionality?.openAsExpandable) && (
+              <Table.Cell width="80"></Table.Cell>
             )}
-            {internalConfig.columns.map((attribute: string) => (
+            {config.columns.map((attribute: string) => (
               <Table.Cell key={attribute}>{attribute}</Table.Cell>
             ))}
-            {internalConfig.editMode && internalConfig.showDelete && (
-              <Table.Cell>Delete</Table.Cell>
+            {functionality?.delete && (
+              <Table.Cell width="80">Delete</Table.Cell>
```

Comment:
This is more of a personal preference, but it might make sense to only show attribute name in headers (and leave this cell empty), since it is very clear from the symbol to understand that the button will do a delete.
You can consider leabing this cell blank :) 

![image](https://github.com/equinor/dm-core-packages/assets/69512295/8574c6aa-159e-46de-9116-4ee8ba559cd4)

Reply:
Yep! I left this as it was, but I also think it makes sense to leave it blank 👍🏻